### ISSUE TITLE
Icon not working

### ISSUE THREAD

**Issue by HarmtH** [hobarrera/caffeine-ng#9]
According to my observation (no icon) and this page:
http://developer.ubuntu.com/api/devel/ubuntu-13.10/python/AppIndicator3-0.1.html
the title and default icon-name are reversed.
Patch below.
```
diff --git a/caffeine/main.py b/caffeine/main.py
index 7a5ba32..5269fb4 100644
--- a/caffeine/main.py
+++ b/caffeine/main.py
@@ -165,8 +165,8 @@ class GUI:
 
         if appindicator_avail:
             self.AppInd = \
-                AppIndicator3.Indicator.new("caffeine-cup-empty",
-                                            "caffeine",
+                AppIndicator3.Indicator.new("caffeine",
+                                            "caffeine-cup-empty",
                                             AppIndicator3.IndicatorCategory.
                                             APPLICATION_STATUS)
```

**Comment by hobarrera** [comment]
It seems to make sense - I really ought to set up a VM with some desktop that uses libindicators to properly test all that.

Have you tested that patch and it works? If so, feel free to send a pull request. Otherwise gimme a couple of days to test this myself.

In either case, thanks for your finding/the patch.

**Comment by HarmtH** [comment]
Yes, I'm using it now and got an icon (Fedora 21 with Gnome Shell). I 
don't have so much experience with GitHub yet, tried to directly push to 
your repo with my account but that got me an access denied :P But I'll 
figure it out.

Harm

**Comment by hobarrera** [comment]
Looks like you figured it out pretty fast. 😉

Closed via #10.
